phpBB/styles/prosilver/template/ucp_profile_profile_info.html

```diff
@@ -13,7 +13,7 @@ <h2>{L_TITLE} <span class="small">[ <a href="{U_USER_PROFILE}" title="{L_VIEW_PR
 	<!-- EVENT ucp_profile_profile_info_before -->
 	<!-- IF S_BIRTHDAYS_ENABLED -->
 		<dl>
-			<dt><label for="bday_day">{L_BIRTHDAY}{L_COLON}</label><br /><span>{L_BIRTHDAY_EXPLAIN}</span></dt>
+			<dt><label for="bday_day">{L_BIRTHDAY}{L_COLON}<!-- EVENT ucp_profile_profile_info_birthday_label_after --></label><br /><span>{L_BIRTHDAY_EXPLAIN}</span></dt>
```

Comment:
Are you sure it should be in the label? If yes, shouldn't it then be called _label_append?

Reply:
Yes, it should be inside the label. The issue author wants to add an asterisk. I tried to make another CPF required and phpBB adds asterisk inside the label. So I believe this is the correct position. I will change the name, though.